File: mapeable/Mapeable.ComparacionesParaPruebasUnitarias/Manejo de propiedades/Propiedad.cs
using System;
namespace ManejoDePropiedades
{
    public class Propiedad
    {
        public string Nombre { get; set; }
        public Type Tipo { get; set; }
        public bool SePuedeEscribir { get; set; }
        public bool SePuedeLeer { get; set; }
        public Type TipoDeLaClase { get; set; }

        public bool EsIgualQue(Propiedad otra)
        {
            bool sonIguales = false;

            if (Nombre.Equals(otra.Nombre) & Tipo.Equals(otra.Tipo))
            {
                sonIguales = true;
            }

            return sonIguales;
        }

    }
}

File: mapeable/Mapeable/Manejo de propiedades/Propiedad.vb
Namespace ManejoDePropiedades
    Public Class Propiedad
        Public Property Nombre() As String
        Public Property Tipo() As Type
        Public Property SePuedeEscribir() As Boolean
        Public Property SePuedeLeer() As Boolean
        Public Property TipoDeLaClase() As Type

        Public Function EsIgualQue(otra As Propiedad) As Boolean
            Dim sonIguales As Boolean = False

            If Nombre.Equals(otra.Nombre) And Tipo.Equals(otra.Tipo) Then
                sonIguales = True
            End If

            Return sonIguales
        End Function
    End Class
End Namespace